tests/chessboard-8px-1bit.rs

```diff
@@ -32,15 +33,49 @@ fn chessboard_8px_1bit_iter() {
     // 8px x 8px image. Check that iterator returns all pixels in it
     assert_eq!(pixels.len(), 8 * 8);
 
+    // Imagemagick inverts using a color mapping table which maps a 0 to [255, 255, 255, 0], hence
+    // this instead of a simple `1` value.
+    let w: u32 = Rgb888::WHITE.into_storage();
+
+    let expected = vec![
+        w, w, 0, 0, w, w, 0, 0, //
+        w, w, 0, 0, w, w, 0, 0, //
+        0, 0, w, w, 0, 0, w, w, //
+        0, 0, w, w, 0, 0, w, w, //
+        w, w, 0, 0, w, w, 0, 0, //
+        w, w, 0, 0, w, w, 0, 0, //
+        0, 0, w, w, 0, 0, w, w, //
+        0, 0, w, w, 0, 0, w, w, //
+    ];
+
+    assert_eq!(pixels, expected);
+}
+
+#[test]
+fn chessboard_8px_1bit_iter_inverted() {
+    // Inverted image created with Imagemagick command:
+    // convert chessboard-8px-1bit.bmp -negate -type bilevel chessboard-8px-1bit-inverted.bmp
+    let bmp = RawBmp::from_slice(include_bytes!("./chessboard-8px-1bit-inverted.bmp"))
+        .expect("Failed to parse");
+
+    let pixels: Vec<u32> = bmp.pixels().map(|pixel| pixel.color).collect();
+
+    // 8px x 8px image. Check that iterator returns all pixels in it
+    assert_eq!(pixels.len(), 8 * 8);
+
+    // Imagemagick inverts using a color mapping table which maps a 0 to [255, 255, 255, 0], hence
+    // this instead of a simple `1` value.
+    let w = Rgb888::WHITE.into_storage();
+
     let expected = vec![
-        1, 1, 0, 0, 1, 1, 0, 0, //
-        1, 1, 0, 0, 1, 1, 0, 0, //
-        0, 0, 1, 1, 0, 0, 1, 1, //
-        0, 0, 1, 1, 0, 0, 1, 1, //
-        1, 1, 0, 0, 1, 1, 0, 0, //
-        1, 1, 0, 0, 1, 1, 0, 0, //
-        0, 0, 1, 1, 0, 0, 1, 1, //
-        0, 0, 1, 1, 0, 0, 1, 1, //
+        0, 0, w, w, 0, 0, w, w, //
+        0, 0, w, w, 0, 0, w, w, //
+        w, w, 0, 0, w, w, 0, 0, //
+        w, w, 0, 0, w, w, 0, 0, //
+        0, 0, w, w, 0, 0, w, w, //
+        0, 0, w, w, 0, 0, w, w, //
+        w, w, 0, 0, w, w, 0, 0, //
+        w, w, 0, 0, w, w, 0, 0, //
```

Comment:
Nit: Also using a `b` variable for black would make this test and the one above a bit easier to understand.